format_vectors.py

```diff
@@ -18,8 +18,12 @@
     "plaintext", "aad", "nonce", "ciphertext",
 ]
 
+ordered_export_keys = [
+    "exporter_context", "L", "exported_value"
+]
+
 encryption_count_keys = [
-    0, 1, 2, 4, 10, 32, 255, 256, 257
+    0, 1, 2, 4,
```

Comment:
Why are you dropping these?  I would at least test octet roll-over, so something like:

```suggestion
    0, 1, 2, 4, 255, 256
```

Reply:
The test vectors don't generate this many (see latest draft).